builtin/rebase--interactive.c

```diff
@@ -160,6 +172,15 @@ int cmd_rebase__interactive(int argc, const char **argv, const char *prefix)
 
 		break;
 	}
+	case SHORTEN_OIDS:
+	case EXPAND_OIDS:
+		return !!transform_todos(flags);
+	case CHECK_TODO_LIST:
+		return !!check_todo_list();
+	case REARRANGE_SQUASH:
+		return !!rearrange_squash();
+	case ADD_EXEC:
+		return !!sequencer_add_exec_commands(cmd);
```

Comment:
In the EDIT_TODO case above there is:

```
ret = edit_todo_list(flags);
break;
```

instead of:

```
return !!edit_todo_list(flags);
```

so it feels that the code could be more consistent by doing things in the same way everywhere.